sdk/storage/azure-storage-blob/tests/_shared/asynctestcase.py

```diff
@@ -27,7 +28,38 @@ def __init__(self):
         return self.token
 
 
+def patch_play_responses(unit_test):
+    """Fixes a bug affecting blob tests by applying https://github.com/kevin1024/vcrpy/pull/511 to vcrpy 3.0.0"""
+
+    try:
+        from vcr.stubs.aiohttp_stubs import _serialize_headers, build_response, Request, URL
+    except ImportError:
+        # return a do-nothing patch when importing from vcr fails
+        return lambda: None
```

Comment:
Does the "do nothing" patch not require args/return values?

Reply:
Patches aren't expected to return anything (https://github.com/Azure/azure-sdk-for-python/blob/master/tools/azure-devtools/src/azure_devtools/scenario_tests/base.py#L147) but they are expected to have a positional parameter! Thanks, will fix.